PyInstaller/hooks/hook-sqlalchemy.py

```diff
@@ -32,3 +34,40 @@
 
     for n in databases:
         hiddenimports.append("sqlalchemy.databases." + n)
+
+
+def hook(hook_api):
+    """
+    SQLAlchemy 0.9 introduced the decorator 'util.dependencies'.  This
+    decorator does imports.  eg:
+
+            @util.dependencies("sqlalchemy.sql.schema")
+
+    This hook scans for included SQLAlchemy modules and then scans those modules
+    for any util.dependencies and marks those modules as hidden imports.
+    """
+
+    if not is_module_satisfies('sqlalchemy >= 0.9'):
+        return
+
+    # this parser is very simplistic but seems to catch all cases as of V1.1
+    depend_regex = re.compile(r'@util.dependencies\([\'"](.*)[\'"]\)')
+
+    hidden_imports_set = set()
+    known_imports = set()
+    for node in hook_api.module_graph.flatten(start=hook_api.module):
+        if isinstance(node, SourceModule):
+            if node.identifier.startswith('sqlalchemy'):
```

Comment:
This will mayl `sqlalchemytool`, too. You want to match either 'sqlalchemy' or start with `sqlalchemy.`.